src/lib/core/datetime/native-date-adapter.spec.ts

```diff
@@ -196,8 +196,11 @@ describe('NativeDateAdapter', () => {
     expect(adapter.parse(date)).not.toBe(date);
   });
 
-  it('should parse invalid value as null', () => {
-    expect(adapter.parse('hello')).toBeNull();
+  it('should parse invalid value as invalid', () => {
+    let d = adapter.parse('hello');
+    expect(d).not.toBeNull();
+    expect(adapter.isDateInstance(d)).toBe(true);
```

Comment:
Nit: add a failure message here? Seems odd without context that 'hello' would be a Date instance. e.g. `Expected string to be fed through Date.parse()`